packages/react/src/components/DatePicker/plugins/fixEventsPlugin.js

```diff
@@ -43,6 +43,8 @@ export default (config) => (fp) => {
     }
   };
 
+  const parse = (dateStr) => fp.parseDate(dateStr, fp.config.dateFormat);
```

Comment:
let's give this function a more intentful name
```suggestion
  const parseDateWithFormat = (dateStr) => fp.parseDate(dateStr, fp.config.dateFormat);
```

Reply:
updated